packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -251,19 +525,54 @@ it('should render custom tick marks as defined by ReactNode.', () => {
   expect(container.querySelector('.custom-tick-mark')).toBeTruthy();
 });
 
+it('should render custom tick marks as defined by ReactNode (vertical)', () => {
+  const { container } = render(
+    verticalSliderWrapper(
+      <Slider
+        values={defaultSingleValue}
+        tickLabels={<span className='custom-tick-mark'>Custom</span>}
+        orientation='vertical'
+      />,
+    ),
+  );
+  assertBaseElement(container);
+  expect(container.querySelector('.custom-tick-mark')).toBeTruthy();
+});
+
 it('should render single track', () => {
   const { container } = render(<Slider values={defaultSingleValue} />);
   assertBaseElement(container);
   expect(container.querySelectorAll('.iui-slider-track').length).toBe(1);
 });
 
+it('should render single track (vertical)', () => {
```

Comment:
Not needed as it is not vertical orientation specific and is already tested. You should check whether other tests are really needed.

Reply:
**[DONE]** Thanks, removed this test